frontend/packages/dev-console/src/components/import/DeployImage.tsx

```diff
@@ -141,6 +150,15 @@ const DeployImage: React.FC<Props> = ({
     healthChecks: healthChecksProbeInitialData,
   };
 
+  const [initialValuesState, setInitialValuesState] = React.useState(initialValues);
+  React.useEffect(() => {
+    if (knScalingConfig) {
+      setInitialValuesState((iv) => {
```

Comment:
nit: maybe we can just call it values as `iv` looks like initial values but it's more previous values